blog/2023-11-06-towards-censorship-resistant-architecture/index.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: towards-censorship-resistant-architecture
+title: Towards a Censorship Resistant Architecture
+authors: [filippopedrazzinfp]
+tags: [llm, self-hosted, prem, open-source, petals, p2p, decentralized-ai]
+description: "Prem - Towards a Censorship Resistant Architecture"
+image: "./banner.png"
+---
+
+![Prem Banner](./banner.png)
+
+> Horizontal Scaling is all you need.
+
+Prem Labs is introducing **Prem Network**, a decentralized AI Platform enabling consumer devices to run Large Language Models without compromising on quality, aggregating the computational power of devices around the globe. Prem Network is a breakthrough in LLM inference deployment as it enables consumer devices to power GPU-intensive models on the edge. Similar to what BitTorrent did for P2P file sharing, Prem Swarm provides AI inference and training for consumer devices.
```

Comment:
@filopedraz is it only for LLM or could it be more general (text-to-image, text-to-video, ...)?

Reply:
For now only LLM.